static/src/javascripts/types/global.d.ts

```diff
@@ -144,7 +149,10 @@ interface Window {
 		queue: Array<() => Promise<void>>;
 		mustardCut?: boolean;
 		polyfilled?: boolean;
+		adBlockers: AdBlockers;
 	};
 
 	confiant?: Confiant;
+
+	googletag?: googletag.Googletag;
```

Comment:
This fixes the typescript errors in `static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.ts`